db/schema.rb

```diff
@@ -699,11 +699,12 @@
     t.string   "invitation_key", :limit=>32, :null=>false, :index=>{:name=>"index_course_user_invitations_on_invitation_key", :unique=>true}
     t.datetime "sent_at"
     t.datetime "confirmed_at"
-    t.integer  "confirmer_id",   :index=>{:name=>"fk__course_user_invitations_confirmer_id"}, :foreign_key=>{:references=>"users", :name=>"fk_course_user_invitations_confirmer_id", :on_update=>:no_action, :on_delete=>:no_action}
     t.integer  "creator_id",     :null=>false, :index=>{:name=>"fk__course_user_invitations_creator_id"}, :foreign_key=>{:references=>"users", :name=>"fk_course_user_invitations_creator_id", :on_update=>:no_action, :on_delete=>:no_action}
     t.integer  "updater_id",     :null=>false, :index=>{:name=>"fk__course_user_invitations_updater_id"}, :foreign_key=>{:references=>"users", :name=>"fk_course_user_invitations_updater_id", :on_update=>:no_action, :on_delete=>:no_action}
     t.datetime "created_at",     :null=>false
     t.datetime "updated_at",     :null=>false
+    t.integer  "confirmer_id",   :index=>{:name=>"fk__course_user_invitations_confirmer_id"}, :foreign_key=>{:references=>"users", :name=>"fk_course_user_invitations_confirmer_id", :on_update=>:no_action, :on_delete=>:no_action}
+    t.integer  "role",           :default=>0
```

Comment:
Our convention is to manually reorder newly columns to the top (based on their meanings), and keep userstamps and timestamps columns and the button.